app/lib/json.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Safely parse a JSON string.
+ * @returns The parsed JSON object or null if parsing fails
+ */
+export const safeJsonParse = (jsonString: string): unknown | null => {
```

Comment:
let's also make it generic, so `export const safeJsonParse = <T = unknown>(jsonString: string): T | null => {`

this way if you provide the type you will have it and if not you will have unknown

Reply:
done